### PR TITLE
[plotly.js] add missing plot data type

### PR DESCRIPTION
**Pull request by wacky**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - https://plot.ly/javascript/pie-charts/
  - https://plot.ly/javascript/radar-chart/
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/plotly.js/index.d.ts [hunk 1]
```diff
@@ -481,7 +481,7 @@ export type ScatterData = PlotData;
 export interface PlotData {
 	type: 'bar' | 'box' | 'candlestick' | 'choropleth' | 'contour' | 'heatmap' | 'histogram' | 'mesh3d' |
 		'ohlc' | 'parcoords' | 'pointcloud' | 'scatter' | 'scatter3d' | 'scattergeo' | 'scattergl' |
-		'scatterternary' | 'surface';
+		'scatterternary' | 'surface' | 'pie' | 'scatterpolar';
```
Comment (jonfreedman) [inline types/plotly.js/index.d.ts]: could you re-order this to maintain the alphabetical order of chart types - it's just easier to scan

**Review by jonfreedman — CHANGES_REQUESTED** [review]

File: types/plotly.js/index.d.ts (hunk 1 above)
Comment (wacky) [reply types/plotly.js/index.d.ts]: thanks for reviewing. I've fixed it.

**Review by jonfreedman — APPROVED** [review]
thanks for the PR

**Review by jonfreedman — APPROVED** [review]

**Comment by rbuckton** [comment]
Thanks for the contribution!

**Pull request merged**